gui-apps/organicmaps/organicmaps-9999.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{7..12} )
+PYTHON_COMPAT=( python3_{7,11,12} )
```

Comment:
Why keep 7?

Reply:
I used `gpy-impl $f -python3_{8..10}` and missed some extreme cases